azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/JavaVersionUtils.java

```diff
@@ -5,125 +5,50 @@
 
 package com.microsoft.azure.maven.webapp.utils;
 
-import com.microsoft.azure.management.appservice.JavaVersion;
-import com.microsoft.azure.maven.webapp.models.JavaVersionEnum;
+import com.microsoft.azure.toolkit.lib.appservice.model.JavaVersion;
 
+import com.microsoft.azure.toolkit.lib.common.exception.AzureToolkitRuntimeException;
 import org.apache.commons.lang3.StringUtils;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
-import java.util.stream.Collectors;
 
 public class JavaVersionUtils {
```

Comment:
Do we need to keep this class? Or we could merge the left method to lib `JavaVersion`

Reply:
it will be finally removed, but not in this PR, orelse compilation fails.